packages/client/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Method signatures of `client.publish` and `stream.publish` have changed: optional `metadata` is given an object instead of positional arguments
+  - new metadata fields: `sequenceNumber` and `msgChainId`
```

Comment:
The field `sequenceNumber` wasn't already included before?

Reply:
The feature was implemented in https://github.com/streamr-dev/network-monorepo/pull/674, but was not included in changelog of that PR